apps/src/templates/progress/ProgressBubble.story.jsx

```diff
@@ -49,6 +49,20 @@ export default storybook => {
             disabled={true}
           />
         )
+      }, {
+        name:'hidden tooltips bubble',
+        description: 'should not have tooltips',
+        story: () => (
+          <ProgressBubble
+            level={{
+              levelNumber: 3,
+              status: LevelStatus.perfect,
+              url: "/foo/bar",
+              icon: "fa-document"
+            }}
+            hideToolTips={true}
+          />
```

Comment:
sneak in a little `disabled={false}` in this Story, and Storybook should be happy again 🙂

Reply:
thanks!